staging/src/k8s.io/component-base/logs/api/v1/options.go

```diff
@@ -93,8 +93,25 @@ func (c *LoggingConfiguration) Validate(featureGate featuregate.FeatureGate, fld
 			}
 		}
 	}
-	if _, err := logRegistry.get(c.Format); err != nil {
+	factory, err := logRegistry.get(c.Format)
+	if err != nil {
 		errs = append(errs, field.Invalid(fldPath.Child("format"), c.Format, "Unsupported log format"))
+	} else if factory != nil {
+		feature := factory.Feature()
+		if feature != "" {
```

Comment:
this seems overly complicated and vulnerable to accidental enablement by default (returning "" effectively makes it stable)

would it be feasible to make formats indicate their stability level when they register, so they *have* to choose a level (alpha/beta/stable)